Add new page Calibration type 10 details (#354)
icub-tech-iit/documentation: docs/robot_calibration_types/standard_calibration_types.md

Regarding iCub 3 and the AMO boards, the AMO magnetic target is subdivided into 64 or 32 sectors. The AMO sensor provides a 14-bit resolution per sector, for a total of 14+6=20 or 14+5=19 bits per revolution resolution, respectively. The `Gearbox_E2J` parameter must be configured with the number of sectors in the corresponding `mechanicals/part-mec.xml` file.

> Further details to how the workflow of the whole calibration type 10 is managed are described at [this page](./amo-encoder-joint-calibration-10-details.md)

Reviewer: ```suggestion
Further details of how the workflow of the whole calibration type 10 is managed are outlined on [this page](./amo-encoder-joint-calibration-10-details.md).
```

Author: Updated and squashed commits
